stablehlo/reference/Scope.h

```diff
@@ -35,26 +37,25 @@ class Scope {
   Scope(const Scope &) = delete;
   Scope &operator=(const Scope &) = delete;
 
-  /// Add the mapping from SSA value (`ssaValue`), defined in a region, to
-  /// its evaluated runtime value (`runtimeValue`).
-  void add(Value ssaValue, Tensor runtimeValue);
+  /// Add the mapping from SSA value (`ssaValue`), defined in a region, to its
+  /// evaluated runtime StablehloValue value (`runtimeValue`).
```

Comment:
Can we keep the previous wording that said "runtime value"? Same below for "runtime values".